autoware.repos

```diff
@@ -76,6 +76,10 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/sample_sensor_kit_launch.git
     version: main
+  sensor_kit/awsim_sensor_kit_launch:
```

Comment:
Currently, we add `external` to non-AWF repositories.

```suggestion
  sensor_kit/external/awsim_sensor_kit_launch:
```

Reply:
Reflected by 
https://github.com/autowarefoundation/autoware/pull/2912/commits/5ebc568264f3ca1112d1d35bd0bf9313239a46fa